cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,18 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Wait for the database to be available...")
+        db_connect = None
+        while not db_connect:
+            try:
+                db_connect = connections["default"].ensure_connection()
+            except OperationalError:
+                self.stdout.write("Database unavailable, waiting 1 second...")
+                time.sleep(1)
```

Comment:
The connection to the database is being attempted in an infinite loop until it is successful. This could potentially lead to an infinite loop if the database is not available. It would be better to limit the number of attempts or to provide a timeout. This can be done by introducing a counter that increments on each failed attempt and breaks the loop when a certain limit is reached.